### PR TITLE
Add xylar as a maintainer

### PR DESCRIPTION
**Pull request by xylar**
Make noarch: python

Remove unneeded host dependencies

Add version constraints on run dependencies, following upstream.

Remove unneeded run dependencies

Remove excessive tests (many are unrelated to this package and
will make updates difficult as APIs for other package change)

Add pip check

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@BrentDorsey, it doesn't look like you've had time to maintain this feedstock.  I'm adding myself as a maintainer and doing a bit of tidying up of the recipe.  Assuming you're okay with me becoming the maintainer, please merge this at your convenience (once tests have passed).

**Review by xhochy — APPROVED** [review]

**Pull request merged**
